config/routes.rb

```diff
@@ -850,7 +850,7 @@
 
   # main routes
 
-  root 'main#home'
+  root to: redirect("/users/sign_in")
```

Comment:
Currently, we have a lot of `redirect_to :root` statements in our codebase when errors are occuring (see e.g. in the lectures controller). In case :root is changed to the sign in page, :root in these statements would have to be replaced by something else, because you would not want to redirect a user that is already logged in to the login page in that case (rather to main/start; while users that are not logged in can be redirected to the new :root). Also, the current implementation seems to have an infinite loop problem: Try a path like lectures/4978593 and you get a redirection cascade.

Reply:
Fixed this now. I direct the user to `main#start`, so that `main#home` is never reached anymore and could be deleted in the future since it does not give any added value (because the welcome and about text was moved to the landing page which now encompasses the greeting as well as the login/signup).

Reply:
Also restyled the "language switcher".